### ISSUE TITLE
(Windows) TypeError: 'PySide2.QtCore.qRegisterResourceData' called with wrong argument types

### ISSUE THREAD

**Issue by Ryan-Roberts-Unity** [dreamworksanimation/usdmanager#33]
Seeing the following error when trying to run USD Manager on Win11 with Python 3.7.4 (Qt.py 1.3.8, PySide2 5.14.1, setuptools 40.8.0):
```
c:\Python374\Scripts>python c:\Python374\Scripts\usdmanager
WARNING:usdmanager.utils:Unable to create AssetResolver - Asset links may not work correctly
Traceback (most recent call last):
  File "c:\Python374\Scripts\usdmanager", line 4, in <module>
    __import__('pkg_resources').run_script('usdmanager==0.15.0', 'usdmanager')
  File "c:\python374\lib\site-packages\pkg_resources\__init__.py", line 666, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "c:\python374\lib\site-packages\pkg_resources\__init__.py", line 1446, in run_script
    exec(code, namespace, namespace)
  File "c:\python374\lib\site-packages\usdmanager-0.15.0-py3.7.egg\EGG-INFO\scripts\usdmanager", line 17, in <module>
    from usdmanager import run
  File "c:\python374\lib\site-packages\usdmanager-0.15.0-py3.7.egg\usdmanager\__init__.py", line 78, in <module>
    from . import highlighter, images_rc, utils
  File "c:\python374\lib\site-packages\usdmanager-0.15.0-py3.7.egg\usdmanager\images_rc.py", line 708, in <module>
    qInitResources()
  File "c:\python374\lib\site-packages\usdmanager-0.15.0-py3.7.egg\usdmanager\images_rc.py", line 703, in qInitResources
    QtCore.qRegisterResourceData(0x01, qt_resource_struct, qt_resource_name, qt_resource_data)
TypeError: 'PySide2.QtCore.qRegisterResourceData' called with wrong argument types:
  PySide2.QtCore.qRegisterResourceData(int, str, str, str)
Supported signatures:
  PySide2.QtCore.qRegisterResourceData(int, bytes, bytes, bytes)
```

This file (usdmanager\images_rc.py) is auto generated: `# Created by: The Resource Compiler for PyQt (Qt v4.8.5)`

I'm working around it by modifying `qt_resource_struct`, `qt_resource_name`, and `qt_resource_data` to be byte arrays instead of strings in the installed images_rc.py file (c:\python374\lib\site-packages\usdmanager-0.15.0-py3.7.egg\usdmanager\images_rc.py). 
e.g. changing
```
qt_resource_struct = "\
\x00\x00\x00\x00\x00\x02\x00\x00\x00\x01\x00\x00\x00\x01\
...
\x00\x00\x00\x5c\x00\x00\x00\x00\x00\x01\x00\x00\x19\x1e\
"
```
to
```
qt_resource_struct = b"\
\x00\x00\x00\x00\x00\x02\x00\x00\x00\x01\x00\x00\x00\x01\
...
\x00\x00\x00\x5c\x00\x00\x00\x00\x00\x01\x00\x00\x19\x1e\
"
```

**Comment by mds-dwa** [comment]
Are you having this issue with the python3 branch?

**Comment by Ryan-Roberts-Unity** [comment]
Good question, I didn't see the Python 3 branch before but now that you mentioned it there it is. Closing.

**Comment by mds-dwa** [comment]
I’m hopeful that solves this particular issue since I think it came up before, but if not please reopen.On Jan 22, 2024, at 8:23 AM, Ryan Roberts ***@***.***> wrote:﻿
Closed #33 as completed.
